spec/factories/document_factory.rb

```diff
@@ -8,5 +8,6 @@
     base_path { title ? "/prefix/#{title.parameterize}" : nil }
     document_type { build(:document_type_schema, path_prefix: "/prefix").id }
     publication_state { "changes_not_sent_to_draft" }
+    creator_id { User.last.id }
```

Comment:
I think if we switch this to be a one to one association we can simplify this rather than needing to pull an id out of the db.